index.html

```diff
@@ -364,15 +364,17 @@
 				<h3>Determining the primary pointer</h3>
 				<div>When firing a pointer event, a pointer is considered primary if:
 					<ul>
-						<li>The pointer represents a mouse device.</li>
+						<li>The pointer represents a primary mouse input.</li>
```

Comment:
Seems like we can simplify this whole section now that the mouse, touch and stylus paragraphs are mostly redundant.  See below.